gooddata-sdk/gooddata_sdk/catalog.py

```diff
@@ -17,99 +17,55 @@
 
 
 class CatalogEntry:
+    def __init__(self, entity: dict[str, Any]) -> None:
+        self._e = entity["attributes"]
+        self._entity = entity
+        self._obj_id = ObjId(self._entity["id"], type=self._entity["type"])
```

Comment:
Is the _obj_id constructed in __init__ instead of a property to achieve obj_id stays unchangable via properties?
If not I would prefer to construct _obj_id using properties

Reply:
intention is to force obj_id unchangable.